src/lib/api/filter_general.ts

```diff
@@ -0,0 +1,45 @@
+import { createClient } from "../client/supabase/server";
+
+export async function filter_general(
+  op: string,
+  column: string,
+  values: string[]
+) {
+  const valid_columns = [
+    "name_org",
+    "pseudonym",
+    "pronouns",
+    "email",
+    "phone",
+    "position",
+    "opt_in_communication",
+  ];
```

Comment:
We should get rid of this in favor of introducing a type or an enum for valid columns in a centralized helper in this codebase.